recipe/meta.yaml

```diff
@@ -48,7 +43,7 @@ test:
     - pytest
     - pytest-xdist
     - mpmath
-{% if ver_major_minor == "1.3" %}
+{% if ver_major_minor == "1.4" %}
```

Comment:
you should also have it for 1.4, and 1.3 right? why does this selector even exist? Are trying to keep the recipe portable between versions of scipy? If so, you should add it to pybind11 as well right?

Reply:
I'm just making edits. I think the point of the selector is to make sure the hacks go away as the version changes so that eventually we don't need them anymore. It is not to keeo the recipe working across versions AFAIK.